GettingStarted.md

```diff
@@ -1,60 +1,66 @@
 This document describes how to prepare the HoloToolkit sourcecode for use
 in your project.
 
-# Preparing the HoloToolkit-Unity Code
+To get started either download the HoloToolkit-Unity Asset Package or grab a copy of this repository and download the entire project.
 
-We'll build the HoloToolkit from source since it's a reasonable assumption 
-that you want the latest and greatest if you are reading this document. To
-do this you need to clone the GitHUb repository at 
+# 1a. Download a HoloToolkit-Unity Asset Package
+You can download the latest unity package from the [Asset Packages](External/HoloToolkit/AssetPackages/) folder
```

Comment:
How about to make this PR simple, you remove all this asset package stuff, we merge it and then design it properly?